### PR TITLE
Update and Upgrade task added

### PR DESCRIPTION
**Pull request by WestonJN**

### CODE REVIEW

File: content/projects/linux/beginner/_index.md [hunk 1]
```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
...
+
+When installing a package you use sudo apt install package_name.
+
+3. Install a specific package called tree;
+`sudo apt install tree`
+
+You might ask what is sudo? Sudo allows you to run programs with the security previledges of another user, it is also referred to as a superuser. It is often used to install, start and stop applications that require root priviledges.
+
+dpkg is a tool to intall, build remove and manage Debian Packages,
+This is how you use dpkg, we will install VS Code in this example.
```
Comment (ry-oc) [inline content/projects/linux/beginner/_index.md]: put 's around 'dpkg'

File: content/projects/linux/beginner/_index.md [hunk 2]
```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
...
 1. https://www.lifewire.com/beginners-guide-to-nano-editor-3859002
+
+### Task 8 Update and Upgrade
+1. Update your system with `sudo apt update`. 
+
+2. Use the `sudo apt upgrade` to apply the updates downloaded and select Y for Yes. 
+
+When installing a package you use sudo apt install package_name.
+
+3. Install a specific package called tree;
```
Comment (ry-oc) [inline content/projects/linux/beginner/_index.md]: this displays as 1. not 3.

File: content/projects/linux/beginner/_index.md [hunk 3]
```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
...
+`sudo apt install tree`
+
+You might ask what is sudo? Sudo allows you to run programs with the security previledges of another user, it is also referred to as a superuser. It is often used to install, start and stop applications that require root priviledges.
+
+dpkg is a tool to intall, build remove and manage Debian Packages,
```
Comment (ry-oc) [inline content/projects/linux/beginner/_index.md]: put 's around 'dpkg'

File: content/projects/linux/beginner/_index.md [hunk 4]
```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
...
+1. Update your system with `sudo apt update`. 
+
+2. Use the `sudo apt upgrade` to apply the updates downloaded and select Y for Yes. 
+
+When installing a package you use sudo apt install package_name.
+
+3. Install a specific package called tree;
+`sudo apt install tree`
+
+You might ask what is sudo? Sudo allows you to run programs with the security previledges of another user, it is also referred to as a superuser. It is often used to install, start and stop applications that require root priviledges.
```
Comment (ry-oc) [inline content/projects/linux/beginner/_index.md]: privileges is spelt privileges not previledges

File: content/projects/linux/beginner/_index.md [hunk 5]
```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
...
+
+dpkg is a tool to intall, build remove and manage Debian Packages,
+This is how you use dpkg, we will install VS Code in this example.
+
+Firstly download VS Code [VS Code Download](https://code.visualstudio.com/download)
+then open your terminal in the current folder of the item you just downloaded, type in 
+sudo dpkg -i filename.deb, you will then enter your password and the package will install.
```
Comment (ry-oc) [inline content/projects/linux/beginner/_index.md]: put 's around 'sudo dpkg -i filename.deb'

File: content/projects/linux/beginner/_index.md [hunk 6]
```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
...
+
+3. Install a specific package called tree;
+`sudo apt install tree`
+
+You might ask what is sudo? Sudo allows you to run programs with the security previledges of another user, it is also referred to as a superuser. It is often used to install, start and stop applications that require root priviledges.
+
+dpkg is a tool to intall, build remove and manage Debian Packages,
+This is how you use dpkg, we will install VS Code in this example.
+
+Firstly download VS Code [VS Code Download](https://code.visualstudio.com/download)
```
Comment (ry-oc) [inline content/projects/linux/beginner/_index.md]: Make this 4.

File: content/projects/linux/beginner/_index.md [hunk 7]
```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
 #### Resources
 
 1. https://www.lifewire.com/beginners-guide-to-nano-editor-3859002
+
+### Task 8 Update and Upgrade
+1. Update your system with `sudo apt update`. 
+
+2. Use the `sudo apt upgrade` to apply the updates downloaded and select Y for Yes. 
+
+When installing a package you use sudo apt install package_name.
```
Comment (ry-oc) [inline content/projects/linux/beginner/_index.md]: put 's around 'sudo apt install package_name'

File: content/projects/linux/beginner/_index.md (hunk 2 above)
Comment (WestonJN) [reply content/projects/linux/beginner/_index.md]: I tried to change it, not sure if it worked, please let me know if it does

File: content/projects/linux/beginner/_index.md (hunk 6 above)
Comment (WestonJN) [reply content/projects/linux/beginner/_index.md]: If 3 works then, i'm assuming 4 will work

File: content/projects/linux/beginner/_index.md (hunk 2 above)
Comment (sheenarbw) [reply content/projects/linux/beginner/_index.md]: Look at it yourself. What does it look like. Check it with your own eyes.

File: content/projects/linux/beginner/_index.md (hunk 6 above)
Comment (sheenarbw) [reply content/projects/linux/beginner/_index.md]: We don't do assumptions here. Just look at it. With your eyes. hugo serve it and check for yourself.

File: content/projects/linux/beginner/_index.md (hunk 2 above)
Comment (WestonJN) [reply content/projects/linux/beginner/_index.md]: I've had a look at it

**Comment by WestonJN** [comment]
Thank you. I appreciate your feedback. So i went back to look at the other information that is on this page and by the looks of things none of the seven tasks explain anything. They are all just commands that need to be copied and executed. Hence i made the content that way. 

Should i maybe post resources with more information?

**Pull request merged**